tasks/db_deploy/test/unit_tests/test_orca_reconcile_sql.py

```diff
@@ -0,0 +1,32 @@
+"""
+Name: test_orca_reconcile_sql.py
+
+Description: Testing library for the orca_reconcile_sql.py.
+"""
+
+import unittest
+from inspect import getmembers, isfunction
+
+from sqlalchemy.sql.elements import TextClause
+
+from install import orca_reconcile_sql
+
+
+class TestOrcaSqlLogic(unittest.TestCase):
```

Comment:
You woll likely need additional tests here to help with some of the logic needed for partitions and other items

Reply:
Tests should include number of times called in the loop and testing failure cases when the wrong name is passed.